### DISCUSSION TITLE
레이캐스트중 도움이 필요합니다.

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#352] [5. Q&A]
타임스크립트를 처음 써보는지라 대체 뭐가 잘못됐는지 모르겠습니다.
![질문용 1](https://user-images.githubusercontent.com/73985764/153328633-2530bbab-17cf-4adc-bf2c-ee27886d1a57.JPG)
처음에는 레이캐스트를 위한 raycasthit을 만들었는데 반복적으로 오류가 나고,
해당 방식으로 바꿨더니 이번엔 undefined라고 하네요.

저거 초기화가 안되서 그런거같은데, 어떤식으로 초기화해야할까요?

**Comment by minjujuu** [comment]
hit을 unref 해주어야 할 것 같아요!
참고할만한 가이드가 있어서 링크 남겨봅니다
https://studio.zepeto.me/kr/guides/using-raycast

**Reply by ted002** [reply]
정말 감사합니다...!
